components/gitpod-db/src/typeorm/entity/db-workspace-instance-usage.ts

```diff
@@ -17,6 +17,21 @@ export class DBWorkspaceInstanceUsage {
     @Index("ind_attributionId")
     attributionId: string;
 
+    @Column("varchar")
+    userId: string;
+
+    @Column("varchar")
+    workspaceId: string;
+
+    @Column("varchar")
+    projectId: string;
+
+    @Column("varchar")
+    workspaceType: string;
+
```

Comment:
`workspaceType` has a type defined for it in `gitpod-protocol` so we should use that here rather than `string`.

https://github.com/gitpod-io/gitpod/blob/a86efd21242fa99c342f98a33b6c989b26fb69f4/components/gitpod-protocol/src/protocol.ts#L697